src/content/guides/transferring-assets.mdx

```diff
@@ -0,0 +1,159 @@
+``` mermaid
+sequenceDiagram
+autonumber
+
+participant Recipient
+participant Centrapay
+participant Integrator's Server
+participant Sender
+
+Recipient->>Centrapay: Create an Asset Transfer Request
+Integrator's Server->>Sender: Send notification
+
+alt
+Sender->>Centrapay: Accept asset transfer request
+else
+Note over Centrapay: Asset Transfer request expires
+end
+
+Sender->>Recipient: Transfer assets
+```
+
+1. The recipient [requests an Asset Transfer](/api/asset-transfers/#request-an-asset-transfer)
+    - `recipientAlias` is an alias for the Centrapay account that creates request. This field accepts the recipient’s bank account number.
+    - `senderAccountId` is an **ID** of the Centrapay account that receives the request.
+
+    <CodePanel title="Request" method="POST" path="/api/asset-transfer-requests">
+      ```bash
+      curl -X POST https://service.centrapay.com/api/asset-transfer-requests \
+        -H "X-Api-Key: <TOKEN>" \
+        -H "Content-Type: application/json" \
+        -d '{
+            "recipientAlias": "12-1234-1234567-123",
+            "senderAccountId": "9EDxUT91TMsUjoqoQeBuLQ",
+            "value": "1000",
+            "assetType": "quartz.nzd.test"
+            }'
+      ```
+    </CodePanel>
+
+2. The integrator’s server optionally sends a notification to the sender to prompt them to accept the request.
+
+3. The sender [accepts](/api/asset-transfers/#accept-an-asset-transfer-request) the Asset Transfer Request. If the sender does not accept the request within 14 days, the request will expire.
+    - `senderName` is the name of sender who sends the assets. It is displayed to the recipient in their activity history.
+
+    <CodePanel title="Request" method="POST" path="/api/asset-transfer-requests/{assetTransferId}/accept">
+      ```bash
+      curl -X POST https://service.centrapay.com/api/asset-transfer-requests/{assetTransferId}/accept \
+        -H "X-Api-Key: <TOKEN>" \
+        -H "Content-Type: application/json" \
+        -d '
+          {
+            "assetId": "YGRo6TYYSxH3js7",
+            "value": "1000",
+            "senderName": "Jane Doe"
+          }'
+      ```
+    </CodePanel>
+
+4. The requested assets are transferred from the sender to the recipient.
+
+## Sending Assets
+
+When a sender initiates an Asset Transfer from Centrapay account, the recipient must claim the transfer to receive the assets.
+
+|   Asset transfer status   |                        Description                                                                                 |
+| --------------------------| ------------------------------------------------------------------------------------------------------------------ |
+| `created`                 | Asset transfer successfully created.                                                                               |
+| `sent`                    | Asset transfer notification (sms, email) was sent to a new user                                                    |
```

Comment:
@Wulingoh Suggestion: You are missing trailing full stops (`.`) in a number of these tables. Suggest adding them in for consistency

```suggestion
| `sent`                    | Asset transfer notification (sms, email) was sent to a new user.                                                   |
```